python/lsst/ts/electrometer/controller.py

```diff
@@ -415,13 +419,14 @@ def parse_buffer(self, response):
         while i < 50000:
             intensity.append(unsorted_values[i])
             time.append(unsorted_values[i + 1])
+            voltage.append(unsorted_values[i + 2])
```

Comment:
Interesting that the variable is called `unsorted_values` but you assume that they actually are sorted. Maybe modify the name of the variable to, for instance, `telemetry`?

Reply:
It was an old variable name from the code I inherited from the previous developer. I'll change it to raw_values.